enderio-base/src/main/java/com/enderio/base/client/renderer/item/IItemOverlayRender.java

```diff
@@ -1,7 +1,15 @@
 package com.enderio.base.client.renderer.item;
 
+import com.mojang.blaze3d.vertex.PoseStack;
 import net.minecraft.world.item.ItemStack;
 
 public interface IItemOverlayRender {
-    void renderOverlay(ItemStack pStack, int pXPosition, int pYPosition);
+    /**
+     *
+     * @param pStack the ItemStack to render
+     * @param pXPosition renderPos X
+     * @param pYPosition renderPos Y
+     * @param poseStack a prefabricated PoseStack moved to x and y as well as the basic blitheight. add 201 to render above count and 199 (and below) for anything below
```

Comment:
Can we maybe store some constants somewhere for these two heights, or are they already defined someplace we can link to? Better than having magic numbers.

Reply:
there are no constants for this available. How about just adding 200 as a constant and people can just add or subtract 1 depending on where to render

Reply:
That would work nicely I think